packages/proxmark3-git/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
+TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="2021.05.10-0d3c2978"
+TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/0d3c2978a6516def4d722cacdb9d651b087df2a1.tar.gz
+TERMUX_PKG_SHA256=702e50af914dbc23ad2bb41de19217dc5408f7ae7181868e005dc1052051355d
+TERMUX_PKG_BUILD_DEPENDS="readline"
+TERMUX_PKG_BUILD_IN_SRC="true"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
+
+termux_step_make() {
+        LDLIBS="-L${TERMUX_PREFIX}/lib" INCLUDES_CLIENT="-I${TERMUX_PREFIX}/include/" \
+        make client -j $TERMUX_MAKE_PROCESSES CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC
```

Comment:
Provide the extra args through `TERMUX_STEP_EXTRA_MAKE_ARGS` instead, and avoid overriding the function.  LDLIBS (if this ond is really required) and INCLUDES_CLIENT can then be exported in termux_step_pre_configure() instead

Reply:
> Provide the extra args through TERMUX_STEP_EXTRA_MAKE_ARGS

`TERMUX_PKG_EXTRA_MAKE_ARGS`

Reply:
Is this the correct way now? I will bring these changes to the regular `proxmark3` package if so.